server/game/event.js

```diff
@@ -27,6 +27,7 @@ class Event {
             return;
         }
 
+        card.game.raiseMergedEvent('onCardSaved', { card: card });
```

Comment:
Per @DukeTax's comment, this should be moved after the card has been marked as saved and removed from the card list. It doesn't *strictly* matter since the event is fired asynchronously and the mark-as-saved is synchronous currently. However, if mark-as-saved is ever changed to be asynchronous, it could introduce a subtle bug (reactions firing before the thing actually happens).